templates/firefox-ios.xliff

```diff
@@ -613,14 +613,34 @@
         <source>New Tab opened</source>
         <note>The label text in the Button Toast for switching to a fresh New Tab.</note>
       </trans-unit>
-      <trans-unit id="Copy Image">
+      <trans-unit id="ContextMenu.CopyImage">
         <source>Copy Image</source>
```

Comment:
This doesn't look like a correct string id (also not sure why  it's changing if the string stays the same)

Reply:
We moved these strings because the menu has had some work done to it. As part of the work I wanted to consolidate the strings into a table. The ID could be more descriptive. Is that the problem? Does `ContextMenu.CopyImageTitle` work better?